include/zrepl_prot.h

```diff
@@ -50,7 +50,9 @@ extern "C" {
 #define	MAX_IP_LEN	64
 #define	TARGET_PORT	6060
 
-#define	ZVOL_OP_FLAG_REBUILD 0x01
+#define	ZVOL_OP_FLAG_NONE		0x00
```

Comment:
I find this unusual. If a field is a set of bits (flags) then it goes without saying that the zero is the default value. This can lead to confusion when default - zero value is regarded as if it was a real flag value.

Reply:
Ok. I agree with you. I will remove this.